about.html

```diff
@@ -4,9 +4,11 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script src="https://kit.fontawesome.com/4166e58f6f.js" crossorigin="anonymous"></script>
+    <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
-    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="./styles.css">
```

Comment:
- [Optional] Please use an indentation of two spaces only in all your documents for readability. This is [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) and also a convention.

- [Optional] Please in future projects, make sure you submit a pull request with all changes on it so that your work can be reviewed thoroughly.